openslides_backend/action/actions/committee/import_meeting.py

```diff
@@ -0,0 +1,241 @@
+import time
+from typing import Any, Dict, Iterable, Optional, Tuple
+
+from ....models.base import model_registry
+from ....models.fields import (
+    BaseTemplateField,
+    GenericRelationField,
+    GenericRelationListField,
+    RelationField,
+    RelationListField,
+)
+from ....models.models import Committee
+from ....shared.exceptions import ActionException, MissingPermission
+from ....shared.filters import FilterOperator
+from ....shared.interfaces.event import EventType
+from ....shared.interfaces.write_request import WriteRequest
+from ....shared.patterns import KEYSEPARATOR, Collection, FullQualifiedId
+from ...action import Action
+from ...util.crypto import get_random_string
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData, ActionResults
+
+
+@register_action("committee.import_meeting")
+class CommitteeImportMeeting(Action):
+    """
+    Action to import a meeting.
+    """
+
+    model = Committee()
+    schema = DefaultSchema(Committee()).get_update_schema(
+        additional_required_fields={"meeting_json": {"type": "object"}}
+    )
+
+    def perform(
+        self, action_data: ActionData, user_id: int, internal: bool = False
+    ) -> Tuple[Optional[WriteRequest], Optional[ActionResults]]:
+        """
+        Simplified entrypoint to perform the action.
+        """
+        self.user_id = user_id
+        self.index = 0
+        for instance in action_data:
+            self.validate_instance(instance)
+            try:
+                self.check_permissions(instance)
+            except MissingPermission as e:
+                msg = f"You are not allowed to perform action {self.name}."
+                e.message = msg + " " + e.message
+                raise e
+            self.index += 1
+        self.index = -1
+
+        instances = self.get_updated_instances(action_data)
+        for instance in instances:
+            instance = self.base_update_instance(instance)
```

Comment:
Similar here: you can remove the for-loop. Also, since you do not use either of `get_updated_instances` or `base_update_instance`, you do not need to call it here.

Reply:
`SingularActionMixin` needs `get_updated_instances` and I need `update_instance`.